README.MD

```diff
@@ -10,6 +10,10 @@ Therefor, be carefull to use this app in a production monitoring environment.
 App dependencies:
 - Splunk Windows TA
 
+## Installation
+The latest verion can be downloaded and installed directly from [Splunkbase](https://splunkbase.splunk.com/app/7580).
+Alternatively it could be downloaded from the Releases page on [Github](https://github.com/fox-it/dissect-add-on-for-splunk/releases).
```

Comment:
```suggestion
Alternatively it can be downloaded from the Releases page on [Github](https://github.com/fox-it/dissect-add-on-for-splunk/releases).
```